docs/source/plugins/index.rst

```diff
@@ -1,1175 +1,117 @@
-
-.. note::
-
-    Your plugins directory must be readable by the FiftyOne server.
-
-.. _plugins-download:
-
-Downloading plugins
--------------------
-
-To download and use an existing plugin, all you need is a URL to
-the plugin packaged as a ZIP archive or a link to a GitHub repository
-containing the source code.
-
-You can then download plugins using any of the methods described below:
-
-.. tabs::
-
-  .. group-tab:: CLI
-
-    .. code-block:: shell
-
-        # Download plugin(s) from a GitHub repository
-        fiftyone plugins download https://github.com/<user>/<repo>[/tree/branch]
-
-        # Download plugin(s) by specifying the GitHub repository details
-        fiftyone plugins download <user>/<repo>[/<ref>]
-
-        # Download specific plugins from a GitHub repository
-        fiftyone plugins download \\
-            https://github.com/<user>/<repo>[/tree/branch] \\
-            --plugin-names <name1> <name2> <name3>
-
-  .. group-tab:: Python
-
-    .. code-block:: python
-        :linenos:
-
-        import fiftyone.plugins as fop
-
-        # Download plugin(s) from a GitHub repository
-        fop.download_plugin("https://github.com/<user>/<repo>[/tree/branch]")
-
-        # Download plugin(b) by specifying the GitHub repository details
-        fop.download_plugin("<user>/<repo>[/<ref>]")
-
-        # Download specific plugins from a GitHub repository
-        fop.download_plugin(url_or_gh_repo, plugin_names=["<name1>", "<name2>"])
-
-.. note::
-
-    A GitHub repository may contain multiple plugins. By default, all plugins
-    are installed, but you can select specific ones if desired as shown above.
-
-.. note::
-
-    You can download plugins from private GitHub repositories that you have
-    access to by providing your GitHub personal access token via the
-    ``GITHUB_TOKEN`` environment variable.
-
-.. _plugins-managing:
-
-Managing plugins
-----------------
-
-You can use the :ref:`fiftyone plugins <cli-fiftyone-plugins>` and
-:ref:`fiftyone operators <cli-fiftyone-operators>` CLI methods to perform a
-variety of plugin-related actions, including:
-
--   Listing the plugins and operators that you've downloaded
--   Printing information about plugins and operators that you've downloaded
--   Downloading plugins
--   Initializing new plugins for local development
--   Installing or ensuring plugin requirements
--   Enabling and disabling plugins
--   Deleting plugins
-
-Configuring plugins
--------------------
-
-You can store system-wide plugin configurations under the plugins key of your
-App config or dataset. This allows for configuring plugins at the application
-wide level or per dataset.
+With plugins, you can add new functionality to the FiftyOne App, create
+integrations with other tools and APIs, render custom panels, and add custom
+buttons to menus. You can even schedule long running tasks from within the App
+that execute on a connected workflow orchestration tool like Apache Airflow.
```

Comment:
I think I originally added the "Airflow" ref here, maybe thats not really needed/is distracting as part of the overview.

Reply:
I cross-linked to delegated operations and externally linked to Airflow docs a number of other places in the docs. I considered adding linking to this intro jingle but decided I'd rather readers stay on the page so they scroll down and see the GIFs and example plugins below